src/main/java/randoop/reflection/ClassLiteralExtractor.java

```diff
@@ -21,22 +25,43 @@ class ClassLiteralExtractor extends DefaultClassVisitor {
   /** Map from a class under test to the literal sequences that appear in it. */
   private MultiMap<ClassOrInterfaceType, Sequence> literalMap;
```

Comment:
I suggest eliminating this data structure, in favor of `scopeToConstantStatistics`.